client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -221,20 +222,42 @@ class EmailProvidersComparison extends React.Component {
 	renderGSuiteDetails() {
 		const { currencyCode, gSuiteProduct, translate } = this.props;
 
+		let title = translate( 'G Suite by Google' );
+		let description = translate(
+			"We've partnered with Google to offer you email, storage, docs, calendars, and more."
+		);
+		let logo = gSuiteLogo;
+		const features = [
+			translate( 'Annual billing' ),
+			translate( 'Send and receive from your custom domain' ),
+			translate( '30GB storage' ),
+			translate( 'Email, calendars, and contacts' ),
+			translate( 'Video calls, docs, spreadsheets, and more' ),
+		];
+		let buttonLabel = translate( 'Add G Suite' );
+
+		if ( config.isEnabled( 'google-workspace-migration' ) ) {
+			title = getGoogleMailServiceFamily();
+			description = translate(
+				'The best way to create, communicate, and collaborate. An integrated workspace that is simple and easy to use.'
```

Comment:
The new copy as well as the logo was taken from the partner brand guidelines.

Reply:
A reference/link would have been nice here, for posterity.

Reply:
That's a good point. Unfortunately those guidelines are not publicly accessible, which is whyI didn't link to them here.